File: Visual Studio Product Team/Visual Studio 2010 SDK Samples/60968-Visual Studio 2010 SDK Samples/VSSDK IDE Sample Combo Box/VB/Guids.vb
'**************************************************************************

'Copyright (c) Microsoft Corporation. All rights reserved.
'This code is licensed under the Visual Studio SDK license terms.
'THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
'ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
'IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
'PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.

'**************************************************************************


Imports Microsoft.VisualBasic
Imports System

Namespace Microsoft.Samples.VisualStudio.ComboBox
	Friend Class GuidList
		Public Const guidComboBoxPkgString As String = "CCE7D0F7-2597-4427-BCCE-16F1766BCBC0"
		Public Const guidComboBoxCmdSetString As String = "4BBE20F5-2329-4141-A004-69AE72FB6D73"

        <System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1823:AvoidUnusedPrivateFields")> _
        Public Shared ReadOnly guidComboBoxPkg As New Guid(guidComboBoxPkgString)
        Public Shared ReadOnly guidComboBoxCmdSet As New Guid(guidComboBoxCmdSetString)
	End Class
End Namespace

File: Visual Studio Product Team/Visual Studio 2013 SDK Samples/[C#]-Visual Studio 2013 SDK Samples/Combo Box - VS 2013/C#/Guids.cs
/***************************************************************************

Copyright (c) Microsoft Corporation. All rights reserved.
THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.

***************************************************************************/

using System;

namespace Microsoft.Samples.VisualStudio.ComboBox
{
    static class GuidList
    {
        public const string guidComboBoxPkgString = "40d9f297-25fb-4264-99ed-7785f8331c94";
        public const string guidComboBoxCmdSetString = "04151d39-35b6-4e53-beee-48df3bb8cfe5";

        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1823:AvoidUnusedPrivateFields")]
        public static readonly Guid guidComboBoxPkg = new Guid(guidComboBoxPkgString);
        public static readonly Guid guidComboBoxCmdSet = new Guid(guidComboBoxCmdSetString);
    };
}